kimchi/src/circuits/expr.rs

```diff
@@ -3122,7 +3126,7 @@ where
                 });
                 res
             }
-            Atom(Cell(v)) => v.text(),
+            Atom(Cell(v)) => v.text(&mut HashMap::new()),
```

Comment:
why isn't the cache in the arg of the function given as arg to the recursive call ?

Reply:
Not the same type, and also, it is unused in any case.